@Types/ITechRecord.d.ts

```diff
@@ -126,6 +126,7 @@ export default interface ITechRecord {
   lettersOfAuth: LettersOfAuth;
   historicPrimaryVrm?: string;
   historicSecondaryVrms?: string[];
+  batchId?: string;
```

Comment:
Is this enough? Does the Joi schema not need updating?

Reply:
You're right, I had missed that, Will update now